packages/sdks/snippets/gen1-remix/app/routes/editable-region.tsx

```diff
@@ -0,0 +1,49 @@
+// routes/advanced-child.tsx
+import { Builder, BuilderComponent, builder } from "@builder.io/react";
+import type { LoaderFunctionArgs } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+import { CustomColumns } from "./components/CustomColumns";
+
+builder.init("ee9f13b4981e489a9a1209887695ef2b");
+
+Builder.registerComponent(CustomColumns, {
+  name: "MyColumns",
+  inputs: [
+    {
+      name: "column1",
+      type: "uiBlocks",
+      broadcast: true,
+      hideFromUI: true,
```

Comment:
i don't think we need `hideFromUI` for `uiBlocks` input types. Test that everything works as expected in the editor (that the UI block doesn't show up as an input in the Content Editor sidebase). If that's the case, let's remove it from everywhere in the snippets (this PR and elsewhere)

Reply:
You resolved this comment but did not remove `hideFromUI`. can you confirm that users explicitly need to set it?